packages/default-theme/components/SwLogin.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="sw-login">
+    <div class="form sw-login__form">
+      <h2 class="sw-login__header">Log in</h2>
+      <SfAlert class="sw-login__alert" v-if="error" type="danger" :message="error" />
+      <SfInput
+        v-model="$v.login.$model"
+        name="login"
+        label="Your login"
+        class="form__input"
+        :valid="!$v.login.$error"
+        :disabled="isLoading"
+        errorMessage="Login is required"
```

Comment:
there should be also `onblur` method which invoked `touch` on this field
and same with other field